projects/angular/src/accordion/accordion-panel.spec.ts

```diff
@@ -268,5 +305,45 @@ describe('ClrAccordionPanel', () => {
       fixture.detectChanges();
       expect(panelElement.querySelector('.clr-accordion-header-has-description')).toBeTruthy();
     });
+
+    it('should allow the aria-level attribute to be set to an explicit value', () => {
+      expect(panelElement.querySelector('[role="heading"]')).toBeNull();
+
+      fixture.componentInstance.headingLevel = 5;
+      fixture.componentInstance.showHeading = true;
+      fixture.detectChanges();
+
+      const panelHeading = panelElement.querySelector('[role="heading"]');
+      expect(panelHeading).not.toBeNull();
+      expect(panelHeading.getAttribute('aria-level')).toBe('5');
+    });
+
+    it('when [clrAccordionPanelHeadingEnabled] is not set at all, heading role should not be present', () => {
+      const fixture: ComponentFixture<TestNoBindingComponent> = TestBed.createComponent(TestNoBindingComponent);
+      fixture.detectChanges();
+
+      const accordionPanel = fixture.debugElement.query(By.directive(ClrAccordionPanel)).nativeElement as HTMLElement;
```

Comment:
```suggestion
      const accordionPanel: HTMLElement = fixture.debugElement.query(By.directive(ClrAccordionPanel)).nativeElement;
```

Type annotations are generally better than type assertions when possible to avoid overriding certain compiler type checks.

Reply:
Corrected in commit: d9b21e4